### PR TITLE
Update gameobject_questitem.md

### PR DESCRIPTION
**Pull request by degenesis**
Correct wording

### CODE REVIEW

File: docs/gameobject_questitem.md [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 Column | Type | Description
 --- | --- | ---
-GameObjectEntry | int(10) unsigned | Entry ID of a gameobject that can be looted only while the related quest is active.
+GameObjectEntry | int(10) unsigned | Entry ID of a gameobject that contains quest items only while the related quest is active.
```
Comment (BarbzYHOOL) [inline docs/gameobject_questitem.md]:
```suggestion
GameObjectEntry | int(10) unsigned | Entry ID of a gameobject that contains quest items when the related quest is active.
```

Comment (degenesis) [reply docs/gameobject_questitem.md]: better, although the rest of them say "only while"

Comment (BarbzYHOOL) [reply docs/gameobject_questitem.md]: well you can edit those too :D but thank you for PRs!!

to edit directly in github you can go here https://github.com/azerothcore/wiki/pull/110/commits then click the "..." and "edit file" and you " Commit directly to the patch-3 branch. "

Comment (degenesis) [reply docs/gameobject_questitem.md]: nice, thanks for the tip

**Pull request merged**